packages/integrations/useAxios/index.ts

```diff
@@ -42,12 +42,12 @@ export function useAxios<T = any>(url: string, ...args: any[]) {
       instance = args[1]
   }
 
-  const response = ref<any>(null) as Ref<AxiosResponse<T> | undefined>
-  const data = ref<any>(undefined) as Ref<T | undefined>
+  const response = shallowRef<AxiosResponse<T>>()
```

Comment:
Should it be `const response = shallowRef<AxiosResponse<T> | undefined>()`?

Reply:
![image](https://user-images.githubusercontent.com/7064956/113698864-e40f2c00-96a2-11eb-9d3f-a207ef75c956.png)
It is since `shallowRef`'s signature is `export declare function shallowRef<T = any>(): Ref<T | undefined>;`

Reply:
It's because you didn't pass the default value

```ts
const response = shallowRef<AxiosResponse<T> | null>(null)
```

Reply:
I was going with the original typing that the functions had. Did you want them to be null instead of undefined?

Reply:
Both work to me